src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/network/PathsSpec.groovy

```diff
@@ -41,11 +43,43 @@ class PathsSpec extends HealthCheckSpecification {
             }
         }
 
-        and: "Cleanup: delete flow"
-        flowHelperV2.deleteFlow(flow.flowId)
+        cleanup:
+        flow && flowHelperV2.deleteFlow(flow.flowId)
+    }
+
+    @Tidy
+    @Tags(LOW_PRIORITY)
+    def "Able to get paths between switches for the LATENCY strategy"() {
+        given: "Two active not neighboring switches"
+        def switchPair = topologyHelper.getNotNeighboringSwitchPair()
+
+        and: "Create a flow to reduce available bandwidth on some path between these two switches"
+        def flow = flowHelperV2.addFlow(flowHelperV2.randomFlow(switchPair))
+
+        when: "Get paths between switches using the LATENCY strategy"
+        def paths = northbound.getPaths(switchPair.src.dpId, switchPair.dst.dpId, null,
+                PathComputationStrategy.LATENCY, 10, null)
+
+        then: "Paths will be sorted by bandwidth (descending order) and then by latency (ascending order)"
```

Comment:
Since we are querying a 'latency' strategy, I would expect the results to be sorted by latency first. was this discussed?

Reply:
it was refactored.
from now on it is sorted by the `latency` field(ascending order) and then by the `bandwidth` field (descending order).